### PR TITLE
Adds swing ui which monitors scijava's tasks

### PR DESCRIPTION
**Pull request by NicoKiaru**
Adds a circular progress bar component to the status bar which  displays the overall progress of all registered tasks created via the scijava task service. When the user clicks on the circular progress bar, a frame containing a table summarizing each task progression is displayed.

Each task is rendered in the table with a custom renderer, displaying a label with the name of the task and its status, a linear progress bar, and, if enabled, a label with an estimation of the remaining time until task completion. The estimation assumes a constant completion speed since the start of the task registration. Additionally, the second column of the table displays a clickable stop icon and which can be used to cancel the task.

Before cancellation, a user confirmation dialog can be enabled or disabled.

Note: the scijava-common version is overriden in order to use the PR https://github.com/scijava/scijava-common/pull/432

### CODE REVIEW

**Comment by ctrueden** [comment]
Thanks @NicoKiaru!

File: pom.xml [hunk 1]
```diff
@@ -109,6 +114,7 @@
 		<object-inspector.version>0.1</object-inspector.version>
 		<scijava-plot.version>0.2.0</scijava-plot.version>
 		<jfreesvg.version>3.2</jfreesvg.version>
+		<scijava-common.version>2.88.0</scijava-common.version>
```
Comment (imagejan) [inline pom.xml:117]: I don't understand how the CI build passes. I didn't find version 2.88.0 deployed anywhere... just SNAPSHOT versions. What did I miss?

Comment (ctrueden) [reply pom.xml:117]: Did you look on the new Sonatype infrastructure? https://s01.oss.sonatype.org/#nexus-search;gav~org.scijava~scijava-common~2.88.0~~

Maybe when you looked it hadn't yet mirrored to Maven Central. Although now it has: https://search.maven.org/search?q=g:org.scijava%20AND%20a:scijava-common%20AND%20v:2.88.0

**Comment by NicoKiaru** [comment]
That was fast, and thanks for noticing the field error!

File: pom.xml (hunk 1 above)
Comment (imagejan) [reply pom.xml:117]: @ctrueden wrote:
> Did you look [on the new Sonatype infrastructure](https://s01.oss.sonatype.org/#nexus-search;gav~org.scijava~scijava-common~2.88.0~~)?

Ah, no, I just checked maven.scijava.org and it wasn't listed anywhere. I'll check directly on s01.oss.sonatype.org as well in the future. Thanks for the hint!

Comment (ctrueden) [reply pom.xml:117]: > I just checked maven.scijava.org and it wasn't listed anywhere.

Right, maven.scijava.org's public group pulls down artifacts from the various proxies only on demand. So if you look via the web interface and something is missing, you can try e.g. `mvn dependency:get -Dartifact=org.scijava:scijava-common:2.88.0`, and then if you refresh the web view, the artifact will then be there.

I'm not actually sure, but things may also filter into the maven.scijava.org web search every time the indexer runs (which IIRC is set to nightly). But if it's newly deployed that day, then it won't have been indexed that way yet.

**Pull request merged**